src/test/java/core/basesyntax/service/impl/FruitTransactionServiceImplTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionServiceImplTest {
+    private static FruitTransactionServiceImpl fruitTransactionService;
+
+    @BeforeAll
+    static void beforeAll() {
+        OperationStrategy operationStrategy = new OperationStrategyImpl();
```

Comment:
It is better to make a class field
```suggestion
        operationStrategy = new OperationStrategyImpl();
```